user/models.go

```diff
@@ -69,7 +69,8 @@ type User struct {
 	Deactivated       bool                   `default:"false" bson:"deactivated" json:"deactivated"`
 	DeactivatedAt     time.Time              `bson:"deactivated_at" json:"deactivated_at"`
 	IsVerified        bool          		 `bson:"isverified" json:"isverified"`
-	Organizations     []string               `bson:"workspaces" json:"workspaces"` // should contain (organization) workspace ids
+	Organizations     []string               `bson:"workspaces" json:"workspaces"` 
+	// should contain (organization) workspace ids
```

Comment:
You are touching another person's work. shey you know 😠

Reply:
Haa

Oga...no vex oo